### ISSUE TITLE
Users should pay an amount that corresponds to the cost of creating an MSA

### ISSUE THREAD

**Issue by dopatraman** [LibertyDSNP/mrc#35]
Is this a duplicate? (Perhaps this should be for the extrinsic for payment by a different account than new MSA owner?)

# Description
There should be a payment for invoking the extrinsic that creates an MSA account.

## Note
Payers should be able to call the payment extrinsic only after proving that they have enough balance to pay for the invocation.

# Acceptance Criteria
- [ ] Place a minimum weight on the MSA creation extrinsic
- [ ] Calculate base cost of transaction via benchmarking
- [ ] Should be in Tokens (Capacity based is future)
- [ ] V2? Transaction costs should be covered by the submitter, but the MSA need not be connected to it
- [ ] V2? Must have a signature proving the private key holder intends to create an MSA

**Comment by wilwade** [comment]
Duplicate of #100
